src/lib/components/composites/paper-components/paper-view/PaperView.svelte

```diff
@@ -120,7 +120,7 @@ Usage:
     <!-- TODO: Set `isBookmarkedDefault` as soon as endpoint is available -->
     <PaperBookmarkButton isBookmarkedDefault={false} {loadingPaperId} />
 </div>
-<main class="flex h-full w-full flex-col gap-5 px-2 py-4">
+<main class="flex h-full w-full flex-col gap-5 px-5">
```

Comment:
```suggestion
<main class="flex h-full w-full flex-col gap-5 px-5 pb-2">
```

I agree that the horizontal padding should be same and that the gap to the navigation bar should be consistent, but removing the vertical padding completely position the decision buttons and navigation buttons very close to the bottom margin. What do you think of keeping the bottom padding?